google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITWriteTest.java

```diff
@@ -163,8 +161,12 @@ public void writeAlreadyExists() {
   @Ignore // TODO(user): Fix this - backend currently accepts empty mutation.
   @Test
   public void emptyWrite() {
-    expectedException.expect(isSpannerException(ErrorCode.INVALID_ARGUMENT));
-    client.write(Arrays.<Mutation>asList());
+    try {
+      client.write(Arrays.<Mutation>asList());
+      fail("");
```

Comment:
nit: Here and below, add "Expected exception" message.

Reply:
Done.